cclib/parser/qchemparser.py

```diff
@@ -772,6 +778,21 @@ def extract(self, inputfile, line):
                     self.norbdisp_beta = min(self.norbdisp_beta, self.nbasis)
                     self.norbdisp_beta_aonames = min(self.norbdisp_beta_aonames, self.nbasis)
 
+            # Finally, versions of Q-Chem greater than 5.1 print all MOs in
```

Comment:
I have access to more fine-grained versions of Q-Chem, so I'm going to try refining this. But really the whole block seems undesirable...